README.md

```diff
@@ -4,34 +4,57 @@ This library implements the most commonly used Conduce API endpoints.  It can be
 
 `api.py` is the majority of the implementation.  `conduce.py` is a command line utility for invoking the API.  It also servers as a representative implementation of the library.
 
+# Prerequisites
+
+Conduce Python API requires that you have installed the following:
+
+- Python 2.7: https://www.python.org/downloads/
+- pip: https://pip.pypa.io/en/stable/installing/
+
 # Installation
 
 To install using pip:
-
 ```
 pip install git+git://github.com/ConduceInc/conduce-python-api
 ```
-
 or clone the repository and run:
+```
+pip install .
+```
+from the installed directory.
 
+# Using the API
+
+Once you have installed the API, you may use it in a python script by importing the `conduce` module:
+```
+import conduce
+```
+Then you may use API functions, like `create_dataset` as follows:
 ```
-python setup.py install
+conduce.api.create_dataset('my-dataset-name', host='app.condcuce.com', user='email@example.com')
 ```
+However, this requires that the user authenticate with their password when the script is executed.  To avoid this, provide an `api-key` argument instead of `user`.  If you would like to learn more about API key creation, take a look at [API Key Creation](https://conduce-conduce-python-api.readthedocs-hosted.com/en/latest/api-key-creation.html).
+
+The most common use for the python API is data ingest.  Data ingest is too large a topic to be covered here, if you'd like to learn more check out [Intro to Data Ingest](https://conduce-conduce-python-api.readthedocs-hosted.com/en/latest/data-ingest.html)
```

Comment:
The wording here feels too casual.  "take a look at", "check out", etc.

Reply:
I was going for casual.